languages/js/src/enforcement.test.ts

```diff
@@ -3,7 +3,7 @@ import { BaseActor, User, Widget } from '../test/classes';
 import { ForbiddenError, NotFoundError, OsoError } from './errors';
 
 describe(Oso, () => {
-  let oso: Oso<BaseActor, String>;
+  let oso: Oso<BaseActor, string>;
```

Comment:
what effect does this have?

Reply:
Ran afoul of this rule: https://github.com/typescript-eslint/typescript-eslint/blob/9b13a86025860c71c4d805178833ec0902320d06/packages/eslint-plugin/docs/rules/ban-types.md

Looks like [the relevant guidance](https://github.com/typescript-eslint/typescript-eslint/blob/9b13a86025860c71c4d805178833ec0902320d06/packages/eslint-plugin/docs/rules/ban-types.md#default-options) is:

> Don't use the upper-case primitive types, you should use the lower-case types for consistency.